vision/cdcgan_mnist/cGAN_mnist.jl

```diff
@@ -186,4 +186,6 @@ function train(; kws...)
 end    
 
 cd(@__DIR__)
```

Comment:
can we remove this `cd`? It's a bit annoying including when a script changes your working directory.  
Better to have to save the outputs using a relative path instead